src/components/Sidebar/styles.ts

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+import { Flexer } from '../Positioning';
+
+export const SearchWrapper = styled(Flexer)`
+  height: ${props => props.theme.header.height};
+  padding: 10px;
+  background-color: ${props => props.theme.palette.neutral.shade0};
+  flex-shrink: 0;
+  border-bottom: solid 2px ${props => props.theme.palette.neutral.shade2};
+  color: ${props => props.theme.palette.neutral.shade3};
```

Comment:
Not an immediate fan of the use of the palette here compared to using separate theme configs.